syncd/syncd_notifications.cpp

```diff
@@ -301,11 +331,13 @@ void process_on_fdb_event(
 
     SWSS_LOG_INFO("fdb event count: %u", count);
 
+    bool sendntf = true;
+
     for (uint32_t i = 0; i < count; i++)
     {
         sai_fdb_event_notification_data_t *fdb = &data[i];
 
-        check_fdb_event_notification_data(*fdb);
+        sendntf &= check_fdb_event_notification_data(*fdb);
```

Comment:
>check_fdb_event_notification_data [](start = 19, length = 33)

If any check failed, should we break the for-loop immediately?

Reply:
i just wanted to print all the failures if there is more than one